mkdocs.yaml

```
@@ -67,6 +67,9 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
 extra:
   generator: false # Disable watermark
+  analytics:
+    provider: google
+    property: G-1Y31YH9H2Q
```

This setup would require a `GOOGLE_ANALYTICS_KEY` github secret present as an environment variable on build, so the docker compose would need to inherit this key during the deployment

```suggestion
analytics:
    provider: google
    property: !ENV GOOGLE_ANALYTICS_KEY
    feedback:
      title: Was this page helpful?
      ratings:
        - icon: material/emoticon-happy-outline
          name: This page was helpful
          data: 1
          note: >-
            Thanks for your feedback!
        - icon: material/emoticon-sad-outline
          name: This page could be improved
          data: 0
          note: >-
            Thanks for your feedback! Help us improve this page by opening a
            <a href="https://github.com/datajoint/datajoint-company/issues/new?assignees=&labels=question&title=[Feedback]+{title}+-+{url}" target="_blank" rel="noopener">GitHub issue</a>.
```

---

Great idea.  I have confirmed with Drew on the implementation.

---

@kabilar This works on [Element DLC page](https://datajoint.com/docs/elements/element-deeplabcut/0.2/) - It might be unique to web deployments but not work locally. Might be worth adding